frontends/web/src/components/sidebar/sidebar.css

```diff
@@ -217,8 +228,14 @@ a.sidebar-active .single img,
 }
 
 @media (max-width: 1199px) {
+    .closeButton {
+        display:block;
+    }
+
     .sidebar {
-        transition: margin-left 0.2s ease;
+        margin-left: -100vw;
+        transition: margin-left 0.3s ease;
+        width: 100vw;
```

Comment:
1199px is still desktop mode, but with hamburger (unfortunatelly, but one day we fix that).

Could you pick a smaller breakpoint that the navigation menu only goes into fullscreen on mobile portrait, but not on small desktop. Ideally landscape would be as it was before, but mobile portrait would be fullscreen.

Reply:
Ok. I agree with you.

We currently don't have an exact "mobile" breakpoint, but I'll use a the smallest breakpoint that is currently used in the project which is <=560px.

Reply:
Changes made, PTAL 🙏